### PR TITLE
GSGGR-167 Seed geoshop sample data with the code and not wtih the database scripts

### PR DESCRIPTION
**Pull request by lanseg**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -42,6 +42,7 @@ services:
     command: >
       bash -c "
       python3 manage.py migrate &&
+      python3 manage.py seed &&
```
Comment (maltaesousa) [inline docker-compose.yml:45]: This means any existing db would be seeded and we need to override the command for production. While migrate is supposed to be run anytime without harm, imho test data should not be added everytime we migrate, no?

Comment (danduk82) [reply docker-compose.yml:45]: I agree with @maltaesousa ,
maybe add an `docker compose exec` command in the readme to be executed the first time or for testing, but don't put it directly at every startup.

**Review by danduk82 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (lanseg) [reply docker-compose.yml]: Added a note and removed seeding during startup

**Review by danduk82 — APPROVED** [review]
LGTM

**Review by maltaesousa — APPROVED** [review]

**Pull request merged**